Cargo.toml

```diff
@@ -29,7 +29,7 @@ rose_tree = { version ="0.2", optional = true }
 pp-rs = { version = "0.2.1", optional = true }
 
 [features]
-default = []
+default = ["wgsl-in", "wgsl-out"]
```

Comment:
Let's keep it empty 😄

Reply:
I've included this change in this draft PR to make it clear that the other front/back ends are broken